user/troubleshooting/usb-troubleshooting.md

```diff
@@ -62,9 +62,9 @@ Errors suggesting this issue:
 
 - during `qvm-start sys-usb`:
 
-    `
+    ```
     internal error: Unable to reset PCI device [...]  no FLR, PM reset or bus reset available.
-    `
+    ```
```

Comment:
@tokideveloper: FYI, you don't want to indent *and* use triple backticks (```), since this results in the code within the code block starting with spaces. You only want to indent so that it's in line with the rest of the list, not further than that. I've just fixed these four cases: https://github.com/QubesOS/qubes-doc/commit/7060c4a6fa8979ce65082768b3b473145a1b87c7

Reply:
Thanks for the information and fixing!